config/configuration.md

```diff
@@ -75,3 +75,8 @@ API configuration elements
 |online_mode|should mecha start up in online mode?|
 |url|base url for the API|
 |tokenfile|name of the API token file, relative to `certs/`|
+
+# ratsignal_parser
+| Element| description|
+|--------|------------|
+|announcer_nicks| list of case-insensitive nicks, that are allowed to announce incoming clients|
```

Comment:
[error] empty table intentionally empty?

Reply:
404 empty table not found

Reply:
``md

# ratsignal_parser
| Element| description|
|--------|------------|
|announcer_nicks| list of case-insensitive nicks, that are allowed to announce incoming clients|
```

Reply:
nope, i misread here. 🤦‍♂️